tests/conformance/testdata/Dockerfile.heredoc_copy

```diff
@@ -0,0 +1,11 @@
+# syntax=docker/dockerfile:1.3-labs
+FROM busybox
+RUN echo hello
+COPY <<robots.txt <<humans.txt /test/
```

Comment:
Unless there's a syntax reason it's not allowed, we should be testing COPY of a combination of heredocs and items from the context directory somewhere.

Reply:
Maybe also with a `--from` to mix things up a little.

Reply:
Done.